spec/lib/submission_builder/ty2024/states/nj/nj_return_xml_spec.rb

```diff
@@ -84,7 +78,6 @@
         it "populates line 7 XML fields" do
           expect(xml.at("Exemptions YouOver65").text).to eq("")
           expect(xml.at("Exemptions SpouseCuPartner65OrOver").text).to eq("")
-          expect(xml.at("Exemptions Line7").text).to eq("0")
```

Comment:
Why are all the calculated value tests being removed from this file?

Reply:
Line7 and Line6 are not fields supported in the XML Schema, only the checkboxes are.